packages/snap-preact-components/src/components/Molecules/Slideout/Slideout.stories.tsx

```diff
@@ -64,13 +64,13 @@ export default {
 			control: { type: 'text' },
 		},
 		buttonContent: {
-			defaultValue: 'click me',
+			defaultValue: '<div className="ss__slideout__button">click me</div>',
```

Comment:
Should this be JSX now and not a string?

Reply:
<img width="996" alt="image" src="https://user-images.githubusercontent.com/5060400/198400599-15298461-7e91-4862-8f1f-468dfc696f84.png">

Reply:
Looks like just removing this line solves things.